src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.enums.Views;
+import duke.task.Task;
+
+/**
+ * TaskList object to store the list of tasks
+ */
+public class TaskList {
+    private ArrayList<Task> tasksList;
```

Comment:
I like how you used the plural form (tasks) when representing a collection of objects as mentioned in the coding guidelines, as I believe omitting plural form is a common mistake when naming variables.